src/core/lib/iomgr/executor/mpmcqueue.cc

```diff
@@ -122,4 +163,27 @@ void* InfLenFIFOQueue::Get(gpr_timespec* wait_time) {
   return PopFront();
 }
 
+size_t InfLenFIFOQueue::num_node() {
+  size_t num = 1024;
```

Comment:
Can we keep num_node as a member variable, rather than calculating it every time?